src/clipboard-copy-element.js.flow

```diff
@@ -0,0 +1,8 @@
+/* @flow */
+
+declare class ClipboardCopyElement extends HTMLElement {
+  get copiedLabel(): ?string;
+  set copiedLabel(value: string): void;
+  get value(): ?string;
+  set value(value: string): void;
+}
```

Comment:
We might also need to `export` the class from the module since Flow doesn't seem to be registering this global type.

```js
const button = document.querySelector('clipboard-copy')
if (button instanceof ClipboardCopyElement) { // Error: Cannot resolve name `ClipboardCopyElement`. 
  button.value = 'hubot'
}
```

After adding `import ClipboardCopyElement from 'clipboard-copy'`, Flow finds the type definition and the type checks pass.